app/eligibility/index.js

```diff
@@ -0,0 +1,9 @@
+const checkEligibility = require('./check-eligibility')
+const updateWaiting = require('./update-waiting')
+const processUnEligible = require('./process-un-eligible')
+
+module.exports = {

```

Comment:
can we move this into the existing `auto-eligibility` folder and maybe under a `lib` sub directory?

Reply:
maybe later we can remove the `auto-` prefix and keep as `eligibility`